src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.HandleDictionary.cs

```diff
@@ -25,7 +25,7 @@ public static partial class JsonSerializer
                 enumerable = (IEnumerable)jsonPropertyInfo.GetValueAsObject(state.Current.CurrentValue);
                 if (enumerable == null)
                 {
-                    // If applicable, we only want to ignore object properties.
+                    // Ignore extension property and non-ClassType.Object properties.
```

Comment:
Something like "Ignore null extension property, and null ClassType.Object properties if  the IgnoreNullValues option is true." is more accurate and understandable.

We should write null Dictionary values, and this comment says we should ignore them.